### PR TITLE
Adding Noam's long-animation-frames snippet

### PR DESCRIPTION
**Pull request by borisschapira**

### CODE REVIEW

File: pages/Interaction/long-animation-frames [hunk 1]
```diff
@@ -0,0 +1,47 @@
+# Long Animation Frames
+
+To determine when long animation frames (LoAF) happen, you can use [PerformanceObserver](https://developer.mozilla.org/docs/Web/API/PerformanceObserver) and register to observe entries of type `long-animation-frame`. This snippet from [@noamr](https://github.com/noamr) provides additional informations computed from the LoAF raw data.
```
Comment (nucliweb) [inline pages/Interaction/long-animation-frames:3]: The URL it's broken

image

**Comment by tunetheweb** [comment]
As LoAF isn't available by default and requires turning on Experimental Web Features (or an Origin Trial from Chrome 115) we should probably note this.

**Comment by tunetheweb** [comment]
Doh! Just saw you added it!

Maybe worth adding a link to the Origin Trial too: https://developer.chrome.com/origintrials/#/view_trial/3935020174414970881

**Comment by borisschapira** [comment]
Thanks, I added a notice before the snipper. @tunetheweb Do you know what impact the Origin Trial is going to have? We won't be able to use this unless the origin is part of the trial, that's it?

**Comment by tunetheweb** [comment]
Yes. Any origin can sign up for a trial (it's free and automatic!) and add a `<meta>` tag with that origin's unique trial token (or HTTP Header) and that will cause Chrome to automatically enable this feature for all users of the site (or only return these for a proportion of your users if you prefer). Probably less useful for a snippet to be honest, but useful if you want to start logging these in the field and feed them back to analytics.

More info here: https://developer.chrome.com/en/docs/web-platform/origin-trials/

**Comment by tunetheweb** [comment]
FYI edited my previous comment to explain it better.

**Review by nucliweb — APPROVED** [review]

**Pull request merged**